PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java

```diff
@@ -10,10 +10,16 @@
 
 public class MapActivity extends Activity {
 
+    /**
+     * The base map from which a Scenario is played
+     */
+
     private DatabaseHandler mDbHandler;
     private OnClickListener onClickListener = new OnClickListener() {
         @Override
         public void onClick(View v) {
```

Comment:
@anubhakushwaha Should I remove this one - The base map from which a Scenario is played?